x-pack/plugins/viz_editor/bar_chart_plugin/bar_chart_plugin.tsx

```diff
@@ -0,0 +1,63 @@
+/*
```

Comment:
This should prob. be an 'xy-axis' editor which just has an additional `displayType: 'line' | 'area' | 'bar'` since they will share basically everything